api-reference/subscribers/bulk-create-subscribers.mdx

```diff
@@ -2,4 +2,57 @@
 openapi: post /v1/subscribers/bulk
 ---
 
-<Snippet file="apikey-warning.mdx" />
\ No newline at end of file
+<Snippet file="apikey-warning.mdx" />
+
+
+<RequestExample>
+
+```javascript Node.js
+import { Novu } from '@novu/node';
+
+const novu = new Novu('<NOVU_API_KEY>');
+
+await novu.subscribers.identify([
+  {
+    subscriberId: "1"
+    firstName: "Pawan";
+    lastName: "Jain";
+    email: "pawan.jain@domain.com";
+    phone: "+1234567890";
+    avatar: "https://gravatar.com/avatar/553b157d82ac2880237566d5a644e5fe?s=400&d=robohash&r=x";
+    locale: "en-US";
+    data: {
+      isDeveloper : true
+      customKey: "customValue"
+    };
+  },
+  {
+    subscriberId: "2"
+    firstName: "John";
+    lastName: "Doe";
+    email: "john.doe@domain.com";
+    phone: "+1234567891";
+    avatar: "https://gravatar.com/avatar/553b157d82ac2880237566d5a644e5fe?s=400&d=robohash&r=x";
+    locale: "en-UK";
+    data: {
+      isDeveloper : false
+      customKey1: "customValue1"
+    };
+  }
+  // more subscribers ...
+])
+```
+
+</RequestExample>
+
+<ResponseExample>
+
+```json Response
+    {
+      updated: [subscriberId1, subscriberId2],
+      created: [subscriberId3, subscriberId4],
+      failed: [{ message: 'some error message', subscriberId: subscriberId5 }, { message: 'some error message', subscriberId: subscriberId6 }],
+    }
```

Comment:
The response example should be the appropriate response returned when the request is successful, **200 OK.**

So this is also not correct. Please try the functionality yourself on the API reference, copy the response you get and add it here.

Reply:
I had a chat with @p-fernandez and he told me it is okay to add it like that here